examples/declarationdir_with_value/BUILD.bazel

```diff
@@ -14,27 +14,27 @@ ts_project(
 )
 
 filegroup(
-    name = "types",
+    name = "declarations",
     srcs = [":transpile"],
-    output_group = "types",
+    output_group = "declarations",
```

Comment:
hm, this breaking change feels unnecessary to me. Even if you prefer the name "declarations" to "types" it's going to cause a lot of churn in userland code

Reply:
Went back to just `types` in the output group. If users want transitive_declarations or transitive_sources they can use js_filegroup instead